src/snowflake/cli/api/identifiers.py

```diff
@@ -142,8 +141,10 @@ def from_identifier_model_v1(cls, model: ObjectIdentifierBaseModel) -> "FQN":
         return fqn.set_database(model.database).set_schema(model.schema_name)
 
     @classmethod
-    def from_identifier_model_v2(cls, model: Identifier) -> "FQN":
+    def from_identifier_model_v2(cls, model) -> "FQN":
```

Comment:
how come this type annotation was removed?

Reply:
There's cyclic import. I will solve it in followup PR